.github/workflows/bootstrap.yaml

```diff
@@ -98,23 +92,23 @@ jobs:
           echo "TEST=${{ matrix.env.TEST }}" >> $GITHUB_ENV
 
       - name: Before Install
-        run: GITHUB_WORKSPACE="${GITHUB_WORKSPACE}/pulp_catdog" .github/workflows/scripts/before_install.sh
+        run: .github/workflows/scripts/before_install.sh
         shell: bash
 
       - name: Install
-        run: GITHUB_WORKSPACE="${GITHUB_WORKSPACE}/pulp_catdog" .github/workflows/scripts/install.sh
+        run: .github/workflows/scripts/install.sh
         env:
           PY_COLORS: '1'
           ANSIBLE_FORCE_COLOR: '1'
         shell: bash
 
       - name: Before Script
         run: |
-          GITHUB_WORKSPACE="${GITHUB_WORKSPACE}/pulp_catdog" .github/workflows/scripts/before_script.sh
-          GITHUB_WORKSPACE="${GITHUB_WORKSPACE}/pulp_catdog" echo "HTTPIE_CONFIG_DIR=$GITHUB_WORKSPACE/.github" >> $GITHUB_ENV
+          .github/workflows/scripts/before_script.sh
+          echo "HTTPIE_CONFIG_DIR=$PWD/.github" >> $GITHUB_ENV
```

Comment:
I think we can remove this since we don't have the config
https://github.com/pulp/pulp_npm/blob/master/.ci/ansible/config.json

when  we used operator on the CI, I don't know why we needed this config for `docs` job not break

Reply:
In some docs builds we try to run the example scripts. I guess it's needed there.